// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

// For explanation, see include/srsran/scheduler/scheduler_feedback_handler.h
message mac_sched_ul_bsr_indication {
    required uint64 timestamp = 1;
    required uint32 cell_index = 2;
    required uint32 ue_index = 3;
    required uint32 crnti = 4;
    required uint32 type = 5;
    // Passing ul_bsr_lcg_report_list is TBD
}

